hazelcast/proxy/topic.py

```diff
@@ -70,6 +71,22 @@ def publish(self, message: MessageType) -> Future[None]:
         request = topic_publish_codec.encode_request(self.name, message_data)
         return self._invoke(request)
 
+    def publish_all(self, messages: typing.Sequence[MessageType]) -> Future[None]:
+        """Publishes a list of messages to all subscribers of this topic
+
+        Args:
+            messages: The message list to be published
+        """
+        try:
+            data_list = []
+            for m in messages:
+                data_list.append(self._to_data(m))
+        except SchemaNotReplicatedError as e:
+            return self._send_schema_and_retry(e, self.publish, messages)
```

Comment:
The retry calls `self.publish` not the `publish_all`

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/d2d7dc0387ac8e704d4f74533cc1f9d24854773e.